src/pages/Register.jsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import BasicInfo from '../components/BasicInfo';
+import Skills from '../components/Skills';
+
+const StyledFieldset = styled.fieldset`
+  display: flex;
+  flex-direction: column;
+  legend {
+    font-size: 3rem;
+    font-weight: 600;
+  }
+  p {
+    font-size: 1rem;
+  }
+  .red {
+    color: red;
+  }
+  label {
+    font-size: 1.5rem;
+    font-weight: 600;
+  }
+  input[type=text] {
+    display: block;
+  }
+  button {
```

Comment:
If you check out the design, most of these styles are reused all over the app.
Can we update the global style or create separate components instead?